std/stdio.d

```diff
@@ -2234,24 +2237,26 @@ $(XREF file,readText)
         }
     }
 
+    ///
     unittest
```

Comment:
Hmm either this was done on purpose, but then the entire method shouldn't be in here?

Otherwise let's add some doc.
Still waiting for the answer of experienced Phobos member about the history of `byRecord`.
It does seem useful...